File: tiq-e/array/1/main.py
from typing import List


class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        l = len(nums)
        s = 0
        for i in range(l):
            if nums[i] != nums[s]:
                s += 1
                nums[s] = nums[i]
        return s + 1


print(Solution().removeDuplicates([1, 2, 3, 3, 3]))

File: tiq-e/array/1/main.rs
struct Solution;

impl Solution {
    pub fn remove_duplicates(nums: &mut Vec<i32>) -> i32 {
        nums.dedup();
        nums.len() as i32
    }
}

fn main() {
    let z = Solution::remove_duplicates(&mut vec![1, 2, 3, 3, 3]);
    println!("{}", z)
}